package.json

```diff
@@ -23,7 +23,7 @@
   "scripts": {
     "build": "NODE_ENV=production polkadot-dev-build-ts",
     "build:zip": "(rm -rf ./master-build.zip && yarn build && cd packages/extension/build && zip -r -FS ../../../master-build.zip *) && (rm -rf ./master-src.zip && zip -r -x '*build*' -x '*node_modules*' -FS ./master-src.zip packages .babelrc.js .editorconfig .eslintignore .eslintrc.js babel.config.js CHANGELOG.md CONTRIBUTING.md jest.config.js lerna.json LICENSE package.json README.md tsconfig.json yarn.lock)",
-    "lint": "eslint --ext .js,.jsx,.ts,.tsx . && tsc --noEmit --pretty",
+    "lint": "eslint --ext .js,.jsx,.ts,.tsx . --fix && tsc --noEmit --pretty",
```

Comment:
Not sure if this should go in there. Of course, it's handy, but it might cause some nasty behavior on the level on CI.
I can recommend two approaches:
1. To switch places of `tsc` and `eslint` and simply allow appending additional flags to `eslint command`
2. Move command to separate `.sh` script and here just run it. In such script managing additional flags, passed as arguments is way easier.